spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/command/TaskScheduleCommandsTest.java

```diff
@@ -35,6 +37,11 @@ public void createSchedule() {
 		schedule().create("schedName", "def", "* * * * *", "", "");
 	}
 
+	@Test
```

Comment:
Can you add one more test to verify that it will fail if a user specifies both --properties and --propertiesFile?